guides/common/modules/ref_compliance-of-rhel-hosts-in-red-hat-insights.adoc

```diff
@@ -0,0 +1,13 @@
+[id="compliance-of-rhel-hosts-in-red-hat-insights_{context}"]
+= Compliance of RHEL Hosts in Red Hat Insights
+
+For RHEL hosts with enabled Insights, you can use the Red Hat Insights compliance service to assess and monitor security policy compliance and vulnerabilities on your hosts.
+
+In this scenario, you define compliance policies in the Insights compliance service on the {RHCloud}, add your hosts to a policy, configure your hosts with the SCAP Security Guide, and scan your hosts by using the Insights client.
+You can use remote execution in {Project} to configure your hosts and scan your hosts on regular basis.
+For more information, see {ManagingHostsDocURL}configuring_and_setting_up_remote_jobs_managing-hosts[Configuring and Setting Up Remote Jobs] in _{ManagingHostsDocTitle}_.
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
```suggestion
.Additional resources

```
Prevents rendering issues downstream.

Reply:
Are you sure? If so, you'd probably want to have a look at _all_ instances of ".Additional resources". Asciidoc syntax should not require an empty line.

Reply:
@adamlazik1 
FWIW, I checked OpenShift docs and found 435 occurrences of ".Additional resources" without an empty line and 743 occurrences with an empty line. Both render correctly. I think there was a rule about empty lines after "Additional resources" a couple years ago. We had to adopt some weird conventions, not required by Asciidoc, because we had to get ready for the Jupiter publishing tool.

Reply:
You are probably right. I guess the rendering issue only occurs if a normal sentence is immediately followed by a bullet list, but since this is a title there should be no issue.